libcxx/test/support/test_offset_time_zone.h

```diff
@@ -0,0 +1,68 @@
+// -*- C++ -*-
+//===----------------------------------------------------------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#ifndef SUPPORT_TEST_OFFSET_TIME_ZONE_H
```

Comment:
I would suggest putting this header in the directory where we test `zoned_time` if we're confident we're not going to use it for something else.